front_end/WorkoutWise/src/App.jsx

```diff
@@ -1,13 +1,64 @@
-import { useState } from 'react'
-// import { Navbar } from "./components/Navbar"
-import { Outlet } from 'react-router-dom'
+import { useState, useEffect } from 'react'
+import  Navbar  from "./components/Navbar"
+import { Link, Outlet, useNavigate } from 'react-router-dom'
 import { HomePage } from './pages/HomePage'
+import { api } from './utilities'
+import { createContext } from 'react'
+
+export const userContext = createContext();
 
 function App() {
 
+  const [user, setUser] = useState(null);
+  const apiKey = import.meta.env.VITE_NINJA_API_KEY;
```

Comment:
Good job, I know we didn't cover this during our Curriculum but you were able to read some docs and get it working. 🤓